surveys/2021-annual-survey/questions.md

```diff
@@ -911,16 +911,63 @@ Select one:
 - Yes
 - No [`NEXT`](#excluding-rust-what-is-your-experience-with-other-kinds-of-programming-languages)
 
-### If you are employed, which category best describes the domain in which *you* work?
+### Which category best describes your current employer's industry?
 
-> TODO - server networking, desktop application, mobile application, web application, embedded, etc.
+Select all that apply:
+
+- Advertising
+- Aerospace
+- Automotive
+- Business software
+- Computer hardware
+- Consumer software
+- Energy
+- Finance
+- Film or Television
+- Gaming
+- Healthcare
+- News media
+- Manufacturing
+- Music
+- Retail
+- Telecommunications
+- Transportation
+- Other (open response)
+
+> **justification**
+>
+> We want to see what industries have what representation in the Rust community.
+> While it's impossible to be precise here, we want to get a general idea.
+> This list is largely adopted from this [Wikipedia article](https://en.wikipedia.org/wiki/Outline_of_industry).
 
 ### Do you write or design software in your work?
 
 - Yes, primarily as an individual contributor (i.e., non-manager).
 - I primarily manage others who do.
 - No [`NEXT`](#excluding-rust-what-is-your-experience-with-other-kinds-of-programming-languages)
 
+### Which categories best describes the tech domain(s) you currently write or design software in?
```

Comment:
Some additional domains that received large responses previously:
- machine learning
- distributed systems
- databases
- data science 
- scientific and numeric computing 
- audio programming
- simulation
- robotics
- security 
- internet of things